src/components/cards/NFTOffersCard.tsx

```diff
@@ -0,0 +1,451 @@
+const NFT_IMAGE_SIZE = 77.75;
+const CARD_HEIGHT = 250;
+
+type SortOption = { name: string; icon: string; criterion: SortCriterion };
+
+const SortOptions = {
+  Highest: {
+    name: i18n.t(i18n.l.nftOffers.card.sortCriteria.highest),
+    icon: '􀑁',
+    criterion: SortCriterion.TopBidValue,
+  },
+  FromFloor: {
+    name: i18n.t(i18n.l.nftOffers.card.sortCriteria.fromFloor),
+    icon: '􀅺',
+    criterion: SortCriterion.FloorDifferencePercentage,
+  },
+  Recent: {
+    name: i18n.t(i18n.l.nftOffers.card.sortCriteria.recent),
+    icon: '􀐫',
+    criterion: SortCriterion.DateCreated,
+  },
+} as const;
+
+const NFTImageMask = () => (
+  <Svg width="77.75" height="77.75" viewBox="0 0 77.75 77.75">
+    <Path
+      d="M77.749 17.9662C77.7477 17.1595 77.747 16.7562 77.6896 16.5228C77.4757 15.6521 77.0923 15.2694 76.2212 15.0569C75.9877 15 75.4085 15 74.25 15V15C68.1749 15 63.25 10.0751 63.25 4V4C63.25 2.84152 63.25 2.26229 63.1931 2.02879C62.9806 1.15771 62.5979 0.774301 61.7272 0.56038C61.4938 0.503038 61.0905 0.502343 60.2838 0.500952C59.7318 0.5 59.1545 0.5 58.55 0.5H19.2C12.4794 0.5 9.11905 0.5 6.55211 1.80792C4.29417 2.9584 2.4584 4.79417 1.30792 7.05211C0 9.61905 0 12.9794 0 19.7V59.05C0 65.7706 0 69.1309 1.30792 71.6979C2.4584 73.9558 4.29417 75.7916 6.55211 76.9421C9.11905 78.25 12.4794 78.25 19.2 78.25H58.55C65.2706 78.25 68.6309 78.25 71.1979 76.9421C73.4558 75.7916 75.2916 73.9558 76.4421 71.6979C77.75 69.1309 77.75 65.7706 77.75 59.05V19.7C77.75 19.0955 77.75 18.5182 77.749 17.9662Z"
+      fill="black"
+    />
+  </Svg>
+);
+
+const NFTImage = ({ url }: { url: string }) => (
+  <ImgixImage
+    source={{ uri: url }}
+    style={{ width: NFT_IMAGE_SIZE, height: NFT_IMAGE_SIZE, borderRadius: 12 }}
+    size={NFT_IMAGE_SIZE}
+  />
+);
+
+const FakeOffer = () => {
+  const { colors } = useTheme();
+  return (
+    <AccentColorProvider color={colors.skeleton}>
+      <Box
+        background="accent"
+        width={{ custom: NFT_IMAGE_SIZE }}
+        height={{ custom: NFT_IMAGE_SIZE }}
+        borderRadius={12}
+      />
+      <Box paddingBottom={{ custom: 7 }} paddingTop={{ custom: 12 }}>
+        <Inline space="4px" alignVertical="center">
+          <Box
+            background="accent"
+            width={{ custom: 12 }}
+            height={{ custom: 12 }}
+            borderRadius={6}
+          />
+          <Box
+            background="accent"
+            width={{ custom: 50 }}
+            height={{ custom: 12 }}
+            borderRadius={6}
+          />
+        </Inline>
+      </Box>
+      <Box
+        background="accent"
+        width={{ custom: 50 }}
+        height={{ custom: 9.3333 }}
+        borderRadius={9.3333 / 2}
+      />
+    </AccentColorProvider>
+  );
+};
+
+const Offer = ({
+  offer,
+  sortCriterion,
+}: {
+  offer: NftOffer;
+  sortCriterion: SortCriterion;
+}) => {
+  const isFloorDiffPercentagePositive = offer.floorDifferencePercentage >= 0;
+  const timeRemaining = offer.validUntil
```

Comment:
`offer.validUntil` may be undefined bc some offers don't expire. Also dealing with potential negative numbers here in case the offer expires during the 1 minute of react query stale time